PraceDomowe/PracaDomowa5/Swiatek_Patryk/Praca_domowa_5.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>Nie o to chodziło, <code>PostId</code>   jest jedynie numerem identyfikującym dany post, nie ma nic wspólnego z liczbą komentarzy dotyczących danego posta. Metoda do narysowania histogramu użyta poprawnie.</p><p> </p><p><code>0.5 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>